webview-ui/src/pages/DeviceViewPage.tsx

```
@@ -150,9 +117,53 @@ function DeviceViewPage() {
+			const onFrame = async (mimeType: string, body: Uint8Array) => {
+				switch (mimeType) {
+					case 'image/jpeg':
+						await onJpegFrame(body);
+						break;
+
+					case 'application/json':
+						await onJsonFrame(body);
+						break;
+
+					default:
+						const bodyText = new TextDecoder().decode(body);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)